docs/apis/url.md

```diff
@@ -105,4 +106,18 @@ apis:
             keys: [net.connectionsReading, net.connectionsWriting, net.connectionsWaiting]
     math:
       net.connectionsDroppedPerSecond: ${net.connectionsAcceptedPerSecond} - ${net.handledPerSecond}
-```
\ No newline at end of file
+```
+
+##  <a name='ReturnResponseHeaders'></a>Include response headers on sample
+
+To include response headers on the metric sample setting `return_headers` attribute to true is needed.
```

Comment:
"To include response headers on the metric sample set `return_headers` attribute to true."

Reply:
👍